decred_wallet/view_controller/TransactionFullDetails/TransactionFullDetailsViewController.swift

```diff
@@ -96,7 +96,12 @@ class TransactionFullDetailsViewController: UIViewController, UITableViewDataSou
     }
     
     @IBAction func viewOnDcrdata(_ sender: Any) {
+       if(UserDefaults.standard.bool(forKey: "walletScanning")){
```

Comment:
Why does this check `walletScanning`? Isn't it supposed to check if app is on testnet mode?